test/package.json

```diff
@@ -34,8 +34,10 @@
     "zombie:test:local": "SH_IMAGE=storage-hub:local pnpm zombienet test configs/simple.zndsl",
     "zombie:test:latest": "SH_IMAGE=docker.io/moonsonglabs/storage-hub:latest pnpm zombienet test configs/simple.zndsl",
     "test:full": "node --no-deprecation --import tsx --test ./suites/zombie/**.spec.ts",
-    "test:bspnet": "NODE_OPTIONS='--no-deprecation' pnpm tsx scripts/checkRunning.ts && node --no-deprecation --test-concurrency 1 --import tsx --test ./suites/integration/**.test.ts",
-    "test:bspnet:only": "NODE_OPTIONS='--no-deprecation' pnpm tsx scripts/checkRunning.ts && node --no-deprecation --import tsx --test-concurrency 1 --test --test-only ./suites/integration/**.test.ts",
+    "test:bspnet": "NODE_OPTIONS='--no-deprecation' pnpm tsx scripts/checkRunning.ts && node --no-deprecation --test-concurrency 1 --import tsx --test ./suites/integration/bsp/**.test.ts",
+    "test:bspnet:only": "NODE_OPTIONS='--no-deprecation' pnpm tsx scripts/checkRunning.ts && node --no-deprecation --import tsx --test-concurrency 1 --test --test-only ./suites/integration/bsp/**.test.ts",
+    "test:user": "NODE_OPTIONS='--no-deprecation' pnpm tsx scripts/checkRunning.ts && node --no-deprecation --test-concurrency 1 --import tsx --test ./suites/integration/user/**.test.ts",
```

Comment:
I decided to split the `bsp` and `user` tests for context. I think we can have one command for all, but wouldn'tit be better to have them separate for local testing purposes?

Reply:
If the main reason is to have easier local testing, I think it's not a good idea. I was under the impression that you can run just a specific test file (or use `.only`).

As we add more test files, where do we stop with adding commands? :)

Reply:
We don't need to add more commands for more test files, I only split those because they have different contexts (user vs bsp). The way I understand it, we could in the future add another folder + command for MSP tests, but it should stop there